src/components/layout/card/style.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import { CardProps } from './index'
+
+const StyledCard = styled<CardProps, 'div'>('div')`
```

Comment:
nit: can we use `export const StyledCard`

Reply:
because if we add one more div to it, then default will not work anymore

Reply:
sure